testgrid/cmd/summarizer/summary/summarizer_service.pb.go

```diff
@@ -0,0 +1,218 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Since we are treating those generated code as "source code", and have others depending on it, (instead of having bazel generating them on the fly), I figure this make more sense to go with the first commit as "source code". Otherwise, test won't run properly.

Let me know if you think all generated code, including the pb.go should go with the second commit and I'll be happy to restructure the commits again. Thanks.

Reply:
Also, would you consider BUILD.bazel as source code or generated? Since they are also generated by running update-bazel.sh. Thanks.

Reply:
Spoke with Erick, I guess we are less concern with having each commit in a buildable state, but putting the generated code in a separate commit will help the reviewer to focus on code that actually needs to be reviewed. Will restructure the commit after all current comments are fixed. Thanks Erick.